nemo/lightning/pytorch/optim/lr_scheduler.py

```diff
@@ -264,21 +264,31 @@ class WarmupAnnealingScheduler(LRSchedulerModule):
 
     def __init__(
         self,
+        warmup_steps: int = 750,
```

Comment:
this will change the behavior of other runs, was it a bug? Do you know if any other model is using this?